examples/frame/src/in_frame/index.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+
+import { root, useInitData } from "@lynx-js/react";
+
+interface MyInitData {
+  id: number;
+  name: string;
+  showDetail: boolean;
+}
+
+interface MyGlobalProps {
+  message: string;
+  status: number;
+  loaded: boolean;
+}
+
+export function App() {
+  let btsInitData = useInitData() as MyInitData;
+  let globalProps = lynx.__globalProps as MyGlobalProps;
```

Comment:
Instead of `as MyGlobalProps`, you should add the type in `rspeedy-env.d.ts` for typed globalProps.